src/http/test/http_test.cpp

```diff
@@ -102,6 +102,32 @@ DOCTEST_TEST_CASE("Parsing error")
   DOCTEST_CHECK(sp.received.empty());
 }
 
+DOCTEST_TEST_CASE("Parsing fuzzing")
+{
+  std::vector<uint8_t> r;
+
+  http::SimpleRequestProcessor sp;
+  http::RequestParser p(sp);
+
+  const auto orig_req = http::build_post_request(r);
```

Comment:
We could parametrise on the verb here, since some of them do have special handling, eg. PRI.

Reply:
This now runs for all verbs that llhttp defines.